_security/configuration/configuration.md

```diff
@@ -139,6 +139,37 @@ If `challenge` is set to `true`, the Security plugin sends a response with statu
 If `challenge` is set to `false` and no `Authorization` header field is set, the Security plugin does not send a `WWW-Authenticate` response back to the client, and authentication fails. You might want to use this setting if you have another challenge `http_authenticator` in your configured authentication domains. One such scenario is when you plan to use basic authentication and OpenID Connect together.
 
 
+## Rate limiting
+
+| Setting | Description |
+| :--- | :--- |
+| `opensearch_security.ui.openid.login.buttonname` |  Display name for the login button. "Log in with single sign-on" by default. |
```

Comment:
Is this change part of this PR?

Reply:
@DarshitChanpura Just in a draft state right now. I'll keep you posted.

Reply:
@DarshitChanpura I believe this is all set. When you get a chance, could you have a last look? Thanks for all the help with this.